lib/rubocop/cop/style/float_division.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for division with integers coerced to floats.
+      # It is recommended to either always use `fdiv` or coerce left side only.
+      #
+      # @example EnforcedStyle: left_coerce (default)
+      #   # bad
+      #   a / b.to_f
+      #   a.to_f / b.to_f
```

Comment:
I think that there is a redundant case that `to_f` is used on both sides. On the other hand, I'm not sure if it is really good to force `to_f` to left side only.

IMO, perhaps the default style of warning only when `to_f` is used on both sides may be convenient. This is a third style.

Reply:
Won't this conflict with what is recommended by styleguide?

```
# bad
a / b.to_f
a.to_f / b.to_f

# good
a.fdiv(b)
a.to_f / b
```

Reply:
Yeah, It conflicts with the current situation. So I'd like to hear other reviewers while exploring the possibility of editing Ruby Style Guide. This is a style that has recently been merged into the style guide and is now verifiable in working code as you implement it ❤️ 

Cc @rubocop-hq/rubocop-core

Reply:
I guess it's fine to have `right_coerce` style. We should the left version in the guide as supposedly it makes it easier to spot what you're trying to do, but obviously that's subjective.

Reply:
I did not have enough explanation. I wanted to show the following option to make a bad case only when both have `to_f`.

```ruby
# bad
a.to_f / b.to_f

# good
a.to_f / b
a / b.to_f
a.fdiv(b)
```

Whether `to_f` is used on the left or right depends on the nature of `a` or `b` parameter. On the other hand, it is redundant that `to_f` is used in both. At that time, I think it is better to let users choose whether to remove the left side `to_f` or the right side `to_f`. This is the reason I recommend this option to default.

Reply:
@koic I have added another option `single_coerce` for this case, but did not make it default because it will be inconsistent with style guide.

Reply:
@tejasbubane The Ruby Style Guide has been updated.
https://rubystyle.guide/#float-division
https://github.com/rubocop-hq/ruby-style-guide/pull/767

Could you update this PR?